### DISCUSSION TITLE
Firmware 1.3.3.36

### DISCUSSION THREAD

**Discussion by dingausmwald** [Guilouz/Creality-Helper-Script-Wiki#681] [General]
Anyone tested this jet?

Newest Version: 1.3.3.36 - 129,14mb

1. Please do a factory reset if you used the root script to install other components, otherwise it might cause errors.
2. Upgrade won't remove fluidd/mainsail, but will overwrite printer configs. Save configs if needed.

Changelog
1. Optimize the logic of filament loading and unloading;
2. Optimize some UI issues;
3. Compatibility with Creality print 5.1;
4. Optimize leveling parameters;
5. Fixed some bugs;

Hope this fixes the bug where the screen freezes on larger prints.

Can't do this atm because they've sent me new left plastic part of the x gantry because one pulley broke after 4 months. Took 5 Days from poland, easy chat through whatsapp. Have to do the repair first. They also have sent me a new toolhead pcb (dono why), new cables and a lidar sensor for free. Just wanted to share this too. Creality support so far ++.

**Comment by Strelka13** [comment]
Installed new firmware. Now can't connect to mainsail.

**Reply by Strelka13** [reply]
![Безымянный](https://github.com/user-attachments/assets/e8224ac1-a4d7-422b-b7a1-0eaf3966be08)

**Reply by dingausmwald** [reply]
Did you do a reset or just the update?

**Reply by Strelka13** [reply]
1) Made update 2) Reset to default 3) Open root 4) Reinstalled mainsail 5) Got error

**Comment by dingausmwald** [comment]
At least that seams to be the wrong order. Reset first, make update, reinstall stuff (optionally make backup before reset to reapply later)
Maybe your router just assigned another IP address to the printer?

**Reply by Strelka13** [reply]
Printer has the same ip adress as before, only MobaXterm said that - "The server identity has changed".

**Reply by Strelka13** [reply]
Jumped back to 1.3.3.29 - everything working good.

**Reply by n8w3rk** [reply]
I was able to install everything normally with the script on the new firmware.

**Reply by Strelka13** [reply]
So tell us please the exact sequence

**Reply by Strelka13** [reply]
Now I made correct reset first, then installed new firmware and reinstalled Mainsail. After got this error.
![2](https://github.com/user-attachments/assets/d94bd4f9-57d9-4eac-807e-f0344de62b66)

**Reply by Strelka13** [reply]
Ohh my God, printer just needed to be rebooted. Now it's connecting. Case closed))) Thank you!

**Reply by n8w3rk** [reply]
Reset via script, update OTA, rooting, reinstall everything via script, but use fluidd
![image](https://github.com/user-attachments/assets/de2b3c0e-a7a4-474c-a73e-d3d1b033c5dd)

**Comment by Strelka13** [comment]
Correct sequence for new firmware installation: 1) Full printer reset 2) Install new firmware 3) Reinstall Mainsail 4) Reboot printer 5) Enjoy

**Comment by feiticeir0** [comment]
Well, it didn't work. 

After the update, I can't get ssh to the printer (K1C) . The port appears filtered . I've done the root access, waited the 30s and still nothing. 

```
Starting Nmap 7.95 ( https://nmap.org ) at 2024-11-01 17:51 WET
Nmap scan report for 192.168.3.172
Host is up (0.014s latency).
Not shown: 996 closed tcp ports (conn-refused)
PORT     STATE    SERVICE
22/tcp   filtered ssh
80/tcp   open     http
8080/tcp open     http-proxy
9999/tcp open     abyss

Nmap done: 1 IP address (1 host up) scanned in 1.38 seconds

```
8080 is MJPEG Streamer... ???
How can I get access to the printer now ?
